doc/reference/network_external.md

```diff
@@ -1,5 +1,15 @@
 # External networks
 
+<!-- Include start external intro -->
+External networks use network interfaces that already exist.
+Therefore, LXD has limited possibility to control them, and LXD features like network ACLs, network forwards and network zones are not supported.
+
+The main purpose for using external networks is to provide an uplink network through a parent interface.
+This external network specifies the presets to use when connecting instances to a parent interface.
+
+LXD supports the following external network types:
```

Comment:
This is slightly extended from what we had in the networks overview originally - please check.